api/v1alpha1/oidc_types.go

```diff
@@ -36,20 +36,21 @@ type OIDC struct {
 	// specified.
 	// +optional
 	Scopes []string `json:"scopes,omitempty"`
+
+	// The redirect URL to be used in the OIDC
+	// [Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest).
+	// If not specified, uses the default redirect URI "%REQ(x-forwarded-proto)%://%REQ(:authority)%/oauth2/callback"
+	RedirectURL *string `json:"redirectURL,omitempty"`
+
+	// The path to sign a user out, clearing their credential cookies.
+	// If not specified, uses a default signout path "/signout"
+	SignoutPath *string `json:"signoutPath,omitempty"`
+
+	// The paths that don't need authentication
+	PassThrough []string `json:"passThrough,omitempty"`
```

Comment:
We need to agree on the API first before moving on.

@arkodg @apjoseph

Reply:
im a -1 on this, this API field is not needed imo, users can create 2 HTTPRoutes - 1 that is targeted by a `BackendTrafficPolicy` and one that is not to satisfy this use case

Reply:
`Passthrough` has been removed. We can revisit and add it later if needed.